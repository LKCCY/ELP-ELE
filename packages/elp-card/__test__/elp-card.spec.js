import { shallowMount } from '@vue/test-utils'
import ElpCard from '../src/elp-card.vue'

describe('card.vue', () => {
  const wrap = shallowMount(ElpCard)
test('快照测试', () => {
    expect(2 + 2).toBe(4)
  })
})
