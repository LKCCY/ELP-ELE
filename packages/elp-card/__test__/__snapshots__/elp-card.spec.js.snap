// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card.vue elp-card快照 1`] = `
<div
  class="elp-card"
>
  <!---->
   
  <main
    class="elp-card__main elp-card__main--symmetry"
  />
</div>
`;
